test/Microsoft.Azure.WebJobs.Host.UnitTests/Blobs/BlobPathSourceTests.cs

```diff
@@ -46,6 +46,27 @@ public void TestToString()
         }
 
         [Fact]
+        public void BlobPath_Throws_OnBackslash()
+        {
+            var exc = Assert.Throws<FormatException>(() => BlobPathSource.Create("container\\blob"));
+            Assert.Contains("Paths must be in the format 'container/blob'", exc.Message);
+        }
+
+        [Fact]
+        public void BlobPath_Throws_OnEmpty()
+        {
+            var exc = Assert.Throws<FormatException>(() => BlobPathSource.Create("/"));
+            Assert.Contains("Paths must be in the format 'container/blob'", exc.Message);
+        }
+
+        [Fact]
+        public void BlobPath_Throws_OnContainerResolves()
```

Comment:
Is this true? 
I know we have this restriction for BlobTriggers, but is this applying it to regular blobs inputs as well?